### PR TITLE
Add ability to cancel/delete builds

### PR DESCRIPTION
**Pull request by jgiardino**
On the Blueprint page > Images tab, adds actions in the list view to cancel/delete builds.

_Includes new text strings_

Actions and flow are different depending on queue_status:

- WAITING
  - action is labeled Stop
  - clicking Stop makes the cancel api call
- RUNNING
  - action is labeled Stop
  - clicking Stop displays a Stop Build modal
  - clicking Stop Build action in the modal makes the cancel api call
- FINISHED
  - action is labeled Delete inside a kebab menu
  - clicking Delete displays a Delete Image modal
  - clicking Delete Image action in the modal makes the delete api call
- FAILED
  - action is labeled Remove
  - clicking Remove makes the delete api call

### CODE REVIEW

**Comment by jgiardino** [comment]
closes #291

File: core/apiCalls.js [hunk 1]
```diff
@@ -167,6 +167,22 @@ export function startComposeApi(blueprintName, composeType) {
   .catch(e => console.log('Error starting compose', e));
 }
 
+export function cancelComposeApi(compose) {
+  return utils.apiFetch(constants.cancel_compose + compose, {
+    method: 'DELETE',
+  }, true)
+  .then(data => data)
```
Comment (larskarlitski) [inline core/apiCalls.js]: This line is redundant. Another one below.

**Comment by larskarlitski** [comment]
Ah, the compose actually failed for me. Please ignore my comment about that. Sorry about the confusion.

**Comment by jgiardino** [comment]
Thanks, @larskarlitski! I made the updates you suggested. Let me know if any other changes are needed.

**Comment by larskarlitski** [comment]
Thank you!

**Comment by jgiardino** [comment]
This is what gets returned when running lorax-test. Is this an example of a flaky test?

[firefox #0-4] 1) Changes Pending Commit Page "after each" hook for "should not commit by clicking Close or X buttons":
[firefox #0-4] Timeout of 90000ms exceeded. Try to reduce the run time or increase your timeout for test specs (http://webdriver.io/guide/testrunner/timeouts.html); if returning a Promise, ensure it resolves. (/end2end/specs/test_pendingCommit.js)
[firefox #0-4] Error: Timeout of 90000ms exceeded. Try to reduce the run time or increase your timeout for test specs (http://webdriver.io/guide/testrunner/timeouts.html); if returning a Promise, ensure it resolves. (/end2end/specs/test_pendingCommit.js)

**Comment by larskarlitski** [comment]
> This is what gets returned when running lorax-test. Is this an example of a flaky test?

Indeed. This failure happens some of the time, for different kinds of tests. It seems unrelated to the changes you've made.

**Pull request merged**